registry/modules/upload-to-s3/router/storeFileS3.ts

```diff
@@ -0,0 +1,80 @@
+import express from "express";
+import { S3Client } from "@aws-sdk/client-s3";
+
+import { storeFileValidator } from "@/schemaValidators/storeFile.zod.js";
+
+import { createStoreFileS3Controller } from "@/modules/upload-to-s3/controllers/storeFileS3.js";
+import { protectedRoute } from "@/modules/auth-basic/middleware/authBasic/jwt.js";
+
+import { response } from "@/modules/shared/utils/response.js";
+
+const s3Client = new S3Client();
+const storeFileController = createStoreFileS3Controller(s3Client);
+const router = express.Router();
+
+router.post("/upload/:fileName", protectedRoute, async (req, res, next) => {
+  const { fileName } = req.params;
+
+  await storeFileValidator()
+    .validateGetFile({ fileName })
+    .then((val) =>
+      storeFileController.uploadFile({ req, fileName: val.fileName })
+    )
+    .then((result) => res.json(response(result)))
+    .catch(next);
+});
+
+router
+  .route("/files")
+  .get(protectedRoute, async (req, res, next) => {
+    const pageToken = req.query.pageToken as string;
+
+    await storeFileValidator()
+      .validateListFiles({ pageToken })
+      .then(storeFileController.getFileList)
+      .then((result) => res.json(response(result)))
+      .catch(next);
+  })
```

Comment:
This exposes the S3 bucket to any authenticated user. Some additional security is missing here. 

Im not sure if we should create a router for the s3 module. Users need to carefully think about what parts of s3 bucket they want to expose.